docker/docker-compose-devnet.yaml

```diff
@@ -21,7 +19,6 @@ services:
     restart: unless-stopped
 
   ipfs:
-    image: ipfs/go-ipfs:v0.5.1
```

Comment:
Leave all this out of the base config since it's now specified in other composed files that are merged in.